src/Discord.Net.WebSocket/Entities/Channels/SocketTextChannel.cs

```diff
@@ -121,6 +124,10 @@ public Task<RestWebhook> GetWebhookAsync(ulong id, RequestOptions options = null
         public Task<IReadOnlyCollection<RestWebhook>> GetWebhooksAsync(RequestOptions options = null)
             => ChannelHelper.GetWebhooksAsync(this, Discord, options);
 
+        // Categories
+        public Task<ICategoryChannel> GetCategoryAsync(RequestOptions options = null)
```

Comment:
Remove this and leave only `INestedChannel.GetCategoryAsync()` since theres already the `Category` property to retrieve it.

Same pattern as how we deal to retrieve users.